packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -44,57 +44,204 @@ const isDropdownSelect = (element = {}) => {
+    /**
+   * Prop to change date when a valid date is selected.
+   */
     adjustDateOnChange: PropTypes.bool,
+    /**
+   * Class name to style the date picker.
+   */
     className: PropTypes.string,
+    /**
+   * Component to render within date picker.
+   */
     children: PropTypes.node,
+    /**
+   * Format of the date selected.
+   */
     dateFormat: PropTypes.oneOfType([
       PropTypes.string,
       PropTypes.array
     ]).isRequired,
+    /**
+   * Prop to style individual days on calebdar.
+   */
     dayClassName: PropTypes.func,
+    /**
+   * Prop to select a dropdown mode for the year dropdown.
+   */
     dropdownMode: PropTypes.oneOf(['scroll', 'select']).isRequired,
+    /**
+   * Maximum Date for a given range.
+   */
     endDate: PropTypes.object,
+    /**
+   * Array to exclude certain dates.
+   */
     excludeDates: PropTypes.array,
+    /**
+   * A callback function to be executed when filter dates is triggered.
+   */
     filterDate: PropTypes.func,
+    /**
+   * Prop Keep the height of calendar dom fixed or variable
+   */
     fixedHeight: PropTypes.bool,
+    /**
+   * A callback function to be executed to format week number .
+   */
     formatWeekNumber: PropTypes.func,
+    /**
+   * Highlight range of dates with custom classes.
+   */
     highlightDates: PropTypes.instanceOf(Map),
+    /**
+   * Show dates only in the given array.
+   */
     includeDates: PropTypes.array,
+    /**
+   * Prop to show inline version of date picker component.
+   */
     inline: PropTypes.bool,
+    /**
+   * Name of locale data for different international formatting.
+   */
     locale: PropTypes.string,
+    /**
+   * Maximum Value of date that can be selected by user.
+   */
     maxDate: PropTypes.object,
+    /**
+   * Minimum Value of date that can be selected by user.
+   */
     minDate: PropTypes.object,
+    /**
+   * Prop to show multiple months on date picker.
+   */
     monthsShown: PropTypes.number,
+    /**
+   * A callback function that is executed when user clicks outside the datepicker.
+   */
     onClickOutside: PropTypes.func,
+    /**
+   * A callback function that is executed when month is selected.
+   */
     onMonthChange: PropTypes.func,
+    /**
+   * Prop to show month navigation.
+   */
     forceShowMonthNavigation: PropTypes.bool,
+    /**
+   * A callback function that is executed when date picker is clicked for dropdown.
+   */
     onDropdownFocus: PropTypes.func,
+    /**
+   * A callback function that is executed when a valid date is selected.
+   */
     onSelect: PropTypes.func.isRequired,
+    /**
+   * A callback function that is executed when a Week number is selected.
+   */
     onWeekSelect: PropTypes.func,
+    /**
+   * A boolean prop to show time picker.
+   */
     showTimeSelect: PropTypes.bool,
+    /**
+   * Prop to choose format of time selected.
+   */
     timeFormat: PropTypes.string,
+    /**
+   * Interval between 2 consecutive times on time picker in minutes.
```

Comment:
What does this mean?

Reply:
> What does this mean?

![image](https://user-images.githubusercontent.com/42207428/61271125-5f3ef900-a7c1-11e9-9802-b40612d90536.png)
The interval between times in here